breathecode/authenticate/tests/urls/test_github_id.py

```diff
@@ -0,0 +1,55 @@
+"""
+Test cases for /user
+"""
+import urllib
+import os
+from django.urls.base import reverse_lazy
+from rest_framework import status
+from ..mixins import AuthTestCase
+# from ..mocks import GithubRequestsMock
+
+
+class AuthenticateTestSuite(AuthTestCase):
+    """Authentication test suite"""
+    def test_github_id_without_url(self):
+        """Test /github without auth"""
+        url = reverse_lazy('authenticate:github_id', kwargs={'user_id':1})
+        response = self.client.get(url)
+
+        data = response.data
+        details = data['details']
+        status_code = data['status_code']
+
+        self.assertEqual(2, len(data))
+        self.assertEqual(details, 'No callback URL specified')
+        self.assertEqual(status_code, status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
A error 500 has not sense

Reply:
Check if exist other test with this error 500 and fix it, because it just can use for server error, it's never use for mistake in the request of client